plan/logical_plan_builder.go

```diff
@@ -442,17 +485,25 @@ func (b *planBuilder) buildProjection(p LogicalPlan, fields []*ast.SelectField,
 				}
 			} else {
 				innerExpr := getInnerFromParentheses(field.Expr)
-				if _, ok := innerExpr.(*ast.ValueExpr); ok && innerExpr.Text() != "" {
-					colName = model.NewCIStr(innerExpr.Text())
-				} else {
-					//Change column name \N to NULL, just when original sql contains \N column
-					fieldText := field.Text()
-					if fieldText == "\\N" {
-						fieldText = "NULL"
+				if valueExpr, ok := innerExpr.(*ast.ValueExpr); ok {
```

Comment:
the logic of setting `colName` and `tblName` is complex enough to be extracted to an individual function